contrib/learn_resource_weights.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+import json
+import logging
+
+logging.basicConfig(format='[%(levelname)s] %(message)s')
+
+
+def parse_commandline():
+    """Parse commandline arguments"""
+    import argparse
```

Comment:
I prefer to have all imports at the top of the file unless there is a good reason not to do so. Imports at the top of the file is also consistent with other Python scripts [[1](https://github.com/CVC4/CVC4/blob/c9e23f66383a4d490aca6d082d40117fe799ee4b/src/options/mkoptions.py), [2](https://github.com/CVC4/CVC4/blob/c9e23f66383a4d490aca6d082d40117fe799ee4b/test/regress/run_regression.py), [3](https://github.com/CVC4/CVC4/blob/c9e23f66383a4d490aca6d082d40117fe799ee4b/test/signatures/run_test.py)].